diff --git a/src/corelib/compat/removed_api.cpp b/src/corelib/compat/removed_api.cpp
index 355d37bdb8..eb9681b5e1 100644
--- a/src/corelib/compat/removed_api.cpp
+++ b/src/corelib/compat/removed_api.cpp
@@ -840,6 +840,12 @@ void qt_qFindChildren_helper(const QObject *parent, const QString &name, const Q
     qt_qFindChildren_helper(parent, QAnyStringView{name}, mo, list, options);
 }
 
+void qt_qFindChildren_helper(const QObject *parent, const QMetaObject &mo,
+                             QList<void*> *list, Qt::FindChildOptions options)
+{
+    qt_qFindChildren_helper(parent, QAnyStringView(), mo, list, options);
+}
+
 QObject *qt_qFindChild_helper(const QObject *parent, const QString &name, const QMetaObject &mo,
                               Qt::FindChildOptions options)
 {
